packages/utils/package.json

```diff
@@ -71,6 +71,7 @@
     "@waku/interfaces": "0.0.20",
     "chai": "^4.3.10",
     "debug": "^4.3.4",
+    "fast-check": "^3.14.0",
```

Comment:
Is it used somewhere? Cannot find it.

Reply:
yes, it's being used in `relay_shard_codec.spec.ts` within `utils/src/common`

Reply:
the file didn't change so it means this was a missing dependency for some reason

nit: let's make it `devDependency` as it is related only for test